File: Old performance Test directories/JavaPerformanceTest/src/INI_Performance_Test.java
import org.ini4j.Ini;

import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;
import java.util.LinkedList;
import java.util.Set;

public class INI_Performance_Test {
    static class IniEntry {
        private String SECTION;
        private String KEY;
        private String VALUE;

        IniEntry(String SECTION, String KEY, String VALUE) {
            this.SECTION = SECTION;
            this.KEY = KEY;
            this.VALUE = VALUE;
        }

        public String getVALUE() {
            return VALUE;
        }

        public void setVALUE(String VALUE) {
            this.VALUE = VALUE;
        }

        public String getKEY() {
            return KEY;
        }

        private void setKEY(String KEY) {
            this.KEY = KEY;
        }

        public String getSECTION() {
            return SECTION;
        }

        private void setSECTION(String SECTION) {
            this.SECTION = SECTION;
        }
    }

    void executeTest() {
        LinkedList<IniEntry> iniEntries = new LinkedList<>();
        try {
            Ini ini = new Ini(new FileReader("../performance_test.ini"));
            Set<String> sections = ini.keySet();
            for (String sectionName: sections) {
                for (int i = 1; i < 6; i++) {
                    iniEntries.add(new IniEntry(sectionName, sectionName+i, ini.get(sectionName, sectionName+i)));
                }
                IniEntry lastEntry = iniEntries.getLast();
                ini.put(lastEntry.getSECTION(), lastEntry.getKEY(), "Testing around");
                ini.store(new FileWriter("../performance_test.ini"));
                ini.put(lastEntry.getSECTION(), lastEntry.getKEY(), lastEntry.getVALUE());
                ini.store(new FileWriter("../performance_test.ini"));
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: Old performance Test directories/KotlinPerformanceTest/src/main/kotlin/INI_Performance_Test.kt
import java.io.FileReader
import java.io.FileWriter
import java.lang.Exception

class INI_Performance_Test {
    data class IniEntry(var SECTION: String, var KEY: String, var VALUE: String)

    fun execute() {
        val iniEntries = mutableListOf<IniEntry>()
        try {
            val ini = Ini(FileReader("performance_test.ini")).run {
                val sections = keySet()
                for (sectionName: String in sections) {
                    for (i in 1..6) {
                        iniEntries += IniEntry(sectionName, sectionName + i, ini.get(sectionName, sectionName + i))
                    }
                    val lastEntry = iniEntries.last()
                    ini.put(lastEntry.SECTION, lastEntry.KEY, "Testing around")
                    ini.store(FileWriter("performance_test.ini"))
                    ini.put(lastEntry.SECTION, lastEntry.KEY, lastEntry.VALUE)
                    ini.store(FileWriter("performance_test.ini"))
                }
            }
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}